packages/libs/react-components/src/styles/stitches.config.ts

```diff
@@ -119,6 +119,10 @@ export const {
     sizes: {
       ...sizes,
     },
+    zIndices: {
+      menu: 100,
```

Comment:
nitpick: might be good to make this a bit more specific like `navigationMenu`

Reply:
makes sense